README.md

```diff
@@ -28,10 +29,8 @@ clevertap.upload(data, {"debug":1, batchSize:50}).then( (res) => {console.log(re
 
 // query for events
 var query = {"event_name":"choseNewFavoriteFood",
-              "props": 
-                [{"name":"value","operator":"contains", "value":"piz"}],
               "from": 20150810,
```

Comment:
20150810 bring this date in 2021. large date range could result in huge data IO. @abhishek-g-clevertap

Reply:
Covered